db/migrate/20210909154907_add_voiding_to_declarations.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddVoidingToDeclarations < ActiveRecord::Migration[6.1]
+  def change
+    add_column :participant_declarations, :voided_at, :datetime
```

Comment:
Should this be in the metadata/profile rather than here?

Reply:
You mean in `profile_declarations`? I don't think so.

Truth be told I am not 100% sure anything should be in `profile_declarations` (why would `profile` need to be there instead of on the declaration itself?) but voiding in particular is a one-way street. We do not plan to ever un-void a declaration. So storing it on the declaration itself seems perfectly fine for me, and it is easier.

Am I missing something?

Reply:
What if support does the "unvoid". I can see the "I voided the wrong declaration by mistake" being a thing. I don't like the idea of anything touching the actual declaration where are they could just delete an incorrect voiding profile.